src/Spd.Manager.Licence/ControllingMemberCrcAppManager.cs

```diff
@@ -280,20 +280,46 @@ private async Task<ChangeSpec> AddDynamicTasks(ControllingMemberCrcApplicationRe
         CancellationToken ct)
     {
         ChangeSpec changes = new();
+        //PeaceOfficerStatusChanged: check if Hold a Position with Peace Officer Status changed, create task with high priority, assign to Licensing CS team
+        PoliceOfficerRoleCode? originalRoleCode = originalApp.PoliceOfficerRoleCode == null ? null
+            : Enum.Parse<PoliceOfficerRoleCode>(originalApp.PoliceOfficerRoleCode.ToString());
 
+        if (newRequest.IsPoliceOrPeaceOfficer != originalApp.IsPoliceOrPeaceOfficer ||
+            newRequest.PoliceOfficerRoleCode != originalRoleCode ||
+            newRequest.OtherOfficerRole != originalApp.OtherOfficerRole ||
+            newFileInfos.Any(d => d.LicenceDocumentTypeCode == LicenceDocumentTypeCode.PoliceBackgroundLetterOfNoConflict))
+        {
+            IEnumerable<string> fileNames = newFileInfos.Where(d => d.LicenceDocumentTypeCode == LicenceDocumentTypeCode.PoliceBackgroundLetterOfNoConflict).Select(d => d.FileName);
+            changes.PeaceOfficerStatusChanged = true;
+            changes.PeaceOfficerStatusChangeTaskId = (await _taskRepository.ManageAsync(new CreateTaskCmd()
+            {
+                Description = $"Business have to requested to update below submitted information for its controlling member:" +
```

Comment:
grammer error in this sentence

Reply:
I still suggest you to ask Kenny to write specific cm update task wording.

Reply:
I copied from confluence, accept that it is not right, I will ask Kenny

Reply:
checked the task creation with Kenny, new modified function should be ok